test/kafka-exactly-once/before-restart.td

```diff
@@ -130,3 +139,59 @@ a  b
 22  11
 3   4
 5   6
+
+$ set sink-schema={
```

Comment:
Wouldn't it work to just use the schema registry and not have the explicit schema here?

Reply:
yup! thanks